scenes/JacobTestScenes/DrainTester.gd

```diff
@@ -0,0 +1,9 @@
+extends Node2D
+
+
+onready var drainable : Drainable = $FullColor
+var drain_radius = 100
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+func _process(delta):
```

Comment:
rest of the PR looks great. only concern here is variable framerate! we can check it out on itch and see if it runs the same. i have a 144hz monitor and ive noticed issues using `_process` before

Reply:
What types of issues are you talking about? I've never noticed issues on my past projects when it came to my 144hz monitor. Are you referring to issues surrounding the difference between the physics update rate nad the process loop?